<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindo</class>
 <widget class="QMainWindow" name="MainWindo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>89</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PEDIDOS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5_novo_pedido">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>440</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Novo Pedido</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_editar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>117</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2_excluir">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>117</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Excluir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3_finalizar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>440</y>
      <width>117</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Finalizar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_pedidos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>761</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Nº do Pedido</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cliente</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Descrição</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data do Pedido</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Valor</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
